### ISSUE TITLE
Storing operators in list fires consistency warning

### ISSUE THREAD

**Issue by jakub-gonet** [rrrene/credo#577]
Credo version: 0.10.0
Elixir version: 1.7.2
OS: Manjaro

In one of my tests I've got this line:
`Enum.to_list(1..25) ++ [&+/2, &-/2, &*/2, &//2]`

I'm getting warning:
```
Consistency       
[C] ↗ There are spaces around operators most of the time, but not here.
```

So, Credo thinks I'm using those Kernel functions as normal operators, but I'm using them as values.

Example:
```elixir
defmodule F do
 def f(), do: 1 + 2
 def g(), do: 3 + 1
 def l(), do: [&+/2, &-/2, &*/2, &//2]
end
```

**Comment by rrrene** [comment]
Thx for reporting! :+1:

I added a test reproducing this issue.
